[2023-12-14T19:09:32.290+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_fda.transform manual__2023-12-14T19:09:02.664876+00:00 [queued]>
[2023-12-14T19:09:32.293+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_fda.transform manual__2023-12-14T19:09:02.664876+00:00 [queued]>
[2023-12-14T19:09:32.293+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-14T19:09:32.298+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-12-14 19:09:02.664876+00:00
[2023-12-14T19:09:32.301+0000] {standard_task_runner.py:57} INFO - Started process 72 to run task
[2023-12-14T19:09:32.304+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'open_fda', 'transform', 'manual__2023-12-14T19:09:02.664876+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/outcome_dag.py', '--cfg-path', '/tmp/tmp0iiic6g4']
[2023-12-14T19:09:32.305+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask transform
[2023-12-14T19:09:32.334+0000] {task_command.py:416} INFO - Running <TaskInstance: open_fda.transform manual__2023-12-14T19:09:02.664876+00:00 [running]> on host b9c035040597
[2023-12-14T19:09:32.369+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_fda' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T19:09:02.664876+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-14T19:09:02.664876+00:00'
[2023-12-14T19:09:33.266+0000] {logging_mixin.py:154} INFO - Tables have been created
[2023-12-14T19:09:33.354+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:314: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2023-12-14T19:09:35.659+0000] {warnings.py:109} WARNING - /opt/***/dags/etl/transformation.py:61: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  new_values[f'{col}_id'] = range(max_id + 1, max_id + 1 + len(new_values))

[2023-12-14T19:09:38.989+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_recall_fct.csv'.
[2023-12-14T19:09:39.939+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_firm_dim.csv'.
[2023-12-14T19:09:40.596+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_status_dim.csv'.
[2023-12-14T19:09:41.254+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_cause_dim.csv'.
[2023-12-14T19:09:41.261+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-14T19:09:41.283+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=open_fda, task_id=transform, execution_date=20231214T190902, start_date=20231214T190932, end_date=20231214T190941
[2023-12-14T19:09:41.351+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-14T19:09:41.369+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
